### PR TITLE
Tidy up code imports

### PR DESCRIPTION
**Pull request by Thurii**
Fixed some off-by-one errors in the markdown code imports.
Completed the tree traversal python implementation.

### CODE REVIEW

File: chapters/tree_traversal/code/python/Tree_example.py [hunk 1]
```diff
@@ -14,12 +14,37 @@ def create_tree(node, num_row, num_child):
...
+
+    if node.data != None:
+        print(node.data)
+
+		
+# This assumes only 2 children, but accounts for other possibilities
+def DFS_recursive_inorder_btree(node)
+    if (len(node.children) == 2)
+        DFS_recursive_inorder_btree(node.children[1])
+        println(n.data)
```
Comment (VikingScientist) [inline chapters/tree_traversal/code/python/Tree_example.py]: I think you mean `print(...)`

File: chapters/tree_traversal/code/python/Tree_example.py [hunk 2]
```diff
@@ -14,12 +14,37 @@ def create_tree(node, num_row, num_child):
...
+def DFS_recursive_postorder(node):
+    for child in node.children:
+		DFS_recursive(child)
+
+    if node.data != None:
+        print(node.data)
+
+		
+# This assumes only 2 children, but accounts for other possibilities
+def DFS_recursive_inorder_btree(node)
```
Comment (VikingScientist) [inline chapters/tree_traversal/code/python/Tree_example.py]: Missing `:`

File: chapters/tree_traversal/code/python/Tree_example.py [hunk 3]
```diff
@@ -14,12 +14,37 @@ def create_tree(node, num_row, num_child):
...
+    if (len(node.children) == 2)
+        DFS_recursive_inorder_btree(node.children[1])
+        println(n.data)
+        DFS_recursive_inorder_btree(node.children[2])
+    elif (len(n.children) == 1)
```
Comment (VikingScientist) [inline chapters/tree_traversal/code/python/Tree_example.py]: `node`, not `n`

File: chapters/tree_traversal/code/python/Tree_example.py [hunk 4]
```diff
@@ -14,12 +14,37 @@ def create_tree(node, num_row, num_child):
...
+        DFS_recursive_inorder_btree(node.children[1])
+        print(node.data)
+    elif (len(n.children) == 0)
```
Comment (VikingScientist) [inline chapters/tree_traversal/code/python/Tree_example.py]: `node`

File: chapters/tree_traversal/code/python/Tree_example.py [hunk 5]
```diff
@@ -34,6 +59,7 @@ def DFS_stack(node):
         for child in temp.children:
             stack.append(child)
 
+
```
Comment (VikingScientist) [inline chapters/tree_traversal/code/python/Tree_example.py]: I don't know if it's established code style, but all other python files use single newline between functions

**Comment by julianschacher** [comment]
@VikingScientist I trust your knowledge.

**Pull request merged**